bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -10,6 +10,27 @@
 
 from ROOT import pythonization
 from libROOTPython import MakeNumpyDataFrame
+# Import numpy lazily (needed for `ndarray` class)
+try:
+    import numpy
```

Comment:
The problem with this is that the import of numpy is no longer lazy, since it will happen every time when a user runs something with PyROOT experimental. 
@stwunsch we can perhaps put this in a separate (helper) python module which is only imported from inside `RDataFrame.AsNumpy`.

Reply:
Uh damn, haven't thought of this :/ Having a separate (sub)module in the experimental PyROOT is fine, but rather complicated for the current PyROOT since we have only `ROOT.py`.

Reply:
I agree. I will start including the new submodule in exp pyroot.

Reply:
For the current PyROOT, we could have `_helpers.py`, sitting next to `ROOT.py` (similarly to `_pythonization.py`).

Reply:
But the release of this feature for the current PyROOT would come with the switch of the experimental one to standard. So there is any use of backporting this and changing the structure of the current PyROOT?

Reply:
`AsNumpy` was already ported to the current PyROOT, so it will not be released when experimental becomes standard, it is already there.